### PR TITLE
Rewrite of SNR function and new search abilities for minor planets and comets.

### PR DESCRIPTION
**Pull request by xscode**
- Reworked the SNR function to hopefully be more representative.
- Added search for Minor planets (Asteroids).
- Added search for bright comets.

### CODE REVIEW

**Comment by xscode** [comment]
The data is in the form of the MPC elements in small text files that are downloaded on first use and computed in real time.  The comet file is also re-downloaded every 7 days to keep it current.

File: front/app.py [hunk 1]
```diff
@@ -2506,6 +2508,107 @@ def on_get(req, resp):
...
+
+    comet = sun + mpc.comet_orbit(row.iloc[0], ts, GM_SUN)
+    Y,M,D,h,m,s = get_UTC_Time()
+    t = ts.utc(Y,M,D,h,m,s)
+    ra, dec, distance = earth.at(t).observe(comet).radec()
+
+    return str(ra).replace(" ",""), str(dec).replace('deg','d').replace("'","m").replace('"',"s").replace(" ",""), comet.target
+
+def searchMinorPlanet(name):
+    with load.open('https://www.minorplanetcenter.net/iau/Ephemerides/Bright/2024/Soft00Bright.txt') as f:
```
Comment (erewhon) [inline front/app.py:2558]: We'll need to bump the URL in a few months, but that's a "future us" problem.

**Comment by erewhon** [comment]
Looks great to me!

File: front/app.py (hunk 1 above)
Comment (xscode) [reply front/app.py:2558]: Problem is they don't automatically release every month, sometimes it takes a while from what I've read.  If I could be more certain on releases I could just use datetime to put in the year.

**Comment by YungKC** [comment]
Love the "keep it current" design, yet good enough for the field.

**Pull request merged**